libs/parallelism/algorithms/tests/regressions/count_3646.cpp

```diff
@@ -16,7 +16,7 @@
 
 #include "iter_sent.hpp"
 
-struct BitCountingIterator : public Iterator<std::int64_t>
+struct BitCountingIterator : public iterator<std::int64_t>
```

Comment:
I realized this wasn't part of the original issue, but would you mind changing `BitCountingIterator` to `bit_counting_iterator`? In general there are a few other names in these files that are in camel case, that should be changed to snake case.

Reply:
I would be happy to do so. Can you please tell me the directory in which these changes are required or is it in the whole repo?

Reply:
It's enough if you do so on the files that you've touched so far in this PR.

Reply:
Sure!

Reply:
There was just that one(`BitCountingIterator`) in the files I have touched.